packages/components/inputs/select-input/src/select-input.story.js

```diff
@@ -106,47 +114,85 @@ storiesOf('Components|Inputs/SelectInputs', module)
+                    'controlShouldRenderValue',
+                    true
+                  )}
+                  id={text('id', '')}
+                  containerId={text('containerId', '')}
+                  isClearable={boolean('isClearable', false)}
+                  isDisabled={boolean('isDisabled', false)}
+                  isReadOnly={boolean('isReadOnly', false)}
+                  isMulti={isMulti}
+                  isSearchable={boolean('isSearchable', false)}
+                  maxMenuHeight={number('maxMenuHeight', 220)}
+                  closeMenuOnSelect={boolean('closeMenuOnSelect', true)}
+                  name={text('name', 'form-field-name')}
+                  onBlur={action('onBlur')}
+                  onChange={(event, ...args) => {
+                    action('onChange')(event, ...args);
+                    onChange(event.target.value);
+                  }}
+                  onFocus={action('onFocus')}
+                  onInputChange={action('onInputChange')}
+                  options={options}
+                  placeholder={text('placeholder', 'Select..')}
+                  tabIndex={text('tabIndex', '0')}
+                  tabSelectsValue={boolean('tabSelectsValue', true)}
+                  value={value}
+                  showOptionGroupDivider={showOptionGroupDivider}
+                  iconLeft={iconLeft ? createElement(iconLeft) : undefined}
+                  menuPortalZIndex={select('menuPortalZIndex', [1, 2, 3], 1)}
+                  // this IIFE is only to make the `menuPortalTarget` knob show up after `menuPortalZIndex`
+                  {...(() => {
+                    const menuPortalTarget = select(
+                      'menuPortalTarget',
+                      ['undefined', 'document.body'],
+                      'undefined'
+                    );
+                    return {
+                      menuPortalTarget:
+                        getMenuPortalTargetValue(menuPortalTarget),
+                    };
+                  })()}
+                />
+                {/* this IIFE is only to make the `menuPortalZIndex-show-neighbouring-stacking-context` knob show up last on the list */}
+                {(() => {
+                  const isActive = boolean(
+                    'menuPortalZIndex-show-neighbouring-stacking-context',
+                    false
+                  );
+                  return (
+                    isActive && (
+                      <div
+                        style={{
+                          width: '300px',
+                          height: '50px',
+                          position: 'relative',
+                          zIndex: 2,
+                        }}
+                      >
+                        <Card theme="dark">
+                          Stacking context with <code>z-index: 2</code>
+                        </Card>
+                      </div>
+                    )
+                  );
+                })()}
```

Comment:
I wanted to keep those knobs close to each other, I don’t know any other way of providing a certain order of knobs. This is the way I thought of.

Also, would you like me to copy this implementation in all other stories where we test components that have this prop?

Reply:
I believe It will also be helpful in other stories.